.github/workflows/unittests.yml

```diff
@@ -34,6 +35,14 @@ jobs:
         shell: bash
 
       - uses: jasp-stats/jasp-actions/setup-test-env@master
+        if: runner.os != 'Windows'
+        with:
+          requiresJAGS: true
+
+      - uses: jasp-stats/jasp-actions/setup-test-env@master
+        if: runner.os == 'Windows'
+        env:
+          JAGS_ROOT: "C:/R-Packages/pkgs/64/JAGS"
```

Comment:
if you revert these changes then it should work again.

Reply:
ok, I'm not sure how to do this properly though, I tried following [this](https://stackoverflow.com/questions/1270514/undoing-a-git-push), with deleting the revinded changes after the first step, but it did not work. And I am scared of experimenting on my own so I do not delete the whole branch


![image](https://user-images.githubusercontent.com/38475991/129905872-e8b5cc0c-62bf-4565-9cca-ffb9ab74d88a.png)

Reply:
Ah sorry, you don't actually have to do a proper revert. you can just delete the changes and then commit that as a new thing. Once this PR is merged, the entire history of this branch gets squashes so it doesn't matter if you do git revert or not.